data/remnant missions.txt

```diff
@@ -495,3 +560,703 @@ mission "Remnant: Technology Available"
+			label end
+			`	After an hour of browsing, the young remnant take off to browse the outfitters and agree to meet you back at the spaceport in an hour.`
+
+
+
+mission "Remnant: Expanded Horizons Quarg 2"
+	name "Quarg Scanning"
+	description "Perform an outfit scan on the Quarg ship <npc>, then land again on <destination>."
+	source
+		planet Wayfarer
+	to offer
+		has "Remnant: Expanded Horizons Quarg 1: done"	
+	on offer
+		conversation
+			`Dawn and her fellow researchers show up a few minutes after you do, excitedly talking about the various things they have seen. "<first>, there is a pair of Quarg ships taking off shortly, and we were hoping you might be able to get scans of them while they are in flight. It would be really helpful for our understanding of how they work."`
+			choice
+				`	"Sure, that sounds simple enough."`
+					accept
+				`	"Sorry, that sounds risky."`
+					defer
+
+	on accept
+		log "Volunteered to show Dawn and some of her fellow researchers the world where humans and Quarg co-exist."
+	
+	npc "scan outfits"
+		government "Quarg"
+		personality staying uninterested target
+		fleet "Quarg"
+		dialog
+			`Dawn and her compatriots eagerly pore over the readouts as they come in, then ask if you could land again to compare their scans to ones taken on the ground.`
+
+	on complete
+		payment 50000
+		conversation
+			`The Remnant researcher's attention to detail is impressive. They carefully quantify every detail of the scans, including notes about the ship's behaviors. By the time you are on the ground again, they have amassed quite a collection of data crystals filled with analysis. You note that they are careful to ensure everything they collect is backed up in multiple places, including what appears to be a small lead lined box.`
+			`	Dawn does some follow-up scans of idle Quarg ships, and her fellow researchers take some time to make sure all their pictures are crisp and detailed. They let you know that they will meet you back in the spaceport bar once they are done.`
+
+
+
+mission "Remnant: Expanded Horizons Quarg 3"
+	name "Expanded Horizons Return"
+	description "Take the Remnant researchers to <planet>"
+	source "Wayfarer"
+	passengers 3
+	blocked `This mission requires bunks for three passengers.`
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Quarg 2: done"
+	on offer
+		conversation
+			`Back in the spaceport bar you wait around for a couple hours. Just when you were about to go find them, the trio walk in looking like they had spent all day on the move, but otherwise unreadable.`
+			`	"OK, I think we are ready to go. Can you take us to <planet>?"`
+				accept
+	on complete
+		payment 50000
```

Comment:
No mention of this payment in the conversation.

Reply:
All the payment stuff has been moved to the end of the mission set.